Xamarin.PropertyEditing.Windows/Themes/Resources.xaml

```diff
@@ -458,6 +458,76 @@
 	<Rectangle x:Shared="False" x:Key="BindingMarker" Style="{StaticResource MarkerStyle}" Fill="{DynamicResource BindingMarkerBrush}" />
 	<Rectangle x:Shared="False" x:Key="SourceMarker" Style="{StaticResource MarkerStyle}" Fill="{Binding Foreground,RelativeSource={RelativeSource AncestorType=local:PropertyButton,Mode=FindAncestor}}" />
 
+	<Style x:Key="RemoveButtonStyle" TargetType="Button">
+		<Setter Property="Margin" Value="5,0,0,0" />
+		<Setter Property="Height" Value="10" />
+		<Setter Property="Width" Value="10" />
+		<Setter Property="VerticalAlignment" Value="Center" />
+		<Setter Property="BorderThickness" Value="0" />
+		<Setter Property="Background" Value="Transparent" />
+		<Setter Property="Foreground" Value="{DynamicResource RemoveElementButtonForegroundBrush}" />
+		<Setter Property="FocusVisualStyle" Value="{DynamicResource GenericVisualFocusStyle}" />
+		<Setter Property="ContentTemplate">
+			<Setter.Value>
+				<DataTemplate>
+					<Grid RenderTransformOrigin="0.5,0.5" Width="6" Height="6" VerticalAlignment="Center" HorizontalAlignment="Center">
+						<Grid.RenderTransform>
+							<RotateTransform Angle="45" />
+						</Grid.RenderTransform>
+
+						<Line X1="0" X2="6" Y1="3" Y2="3" Stroke="{Binding RelativeSource={RelativeSource AncestorType=Button},Path=Foreground}" StrokeThickness="1" />
+						<Line X1="3" X2="3" Y1="0" Y2="6" Stroke="{Binding RelativeSource={RelativeSource AncestorType=Button},Path=Foreground}" StrokeThickness="1" />
```

Comment:
Could you please make the cross larger, so it looks like this?

<img width="437" alt="image" src="https://user-images.githubusercontent.com/4982/58943136-69f87e00-877f-11e9-8372-b1c839751c62.png">

<img width="356" alt="image" src="https://user-images.githubusercontent.com/4982/58943197-898fa680-877f-11e9-90aa-5cdd6480b5d3.png">

Here it is `6.35pt` large. So something around that.

With `8pt` space from the label.

<img width="317" alt="image" src="https://user-images.githubusercontent.com/4982/58943252-a75d0b80-877f-11e9-98ab-6a7e7ff83308.png">